bonus_guides/X_task_supervisor.md

```diff
@@ -0,0 +1,120 @@
+# Running Asynchronous Tasks
+
+Elixir makes concurrency dead simple right out of the box, allowing us to
```

Comment:
I'd remove this first sentence and just start with "There are many instances..."